rfc/README.md

```diff
@@ -0,0 +1,26 @@
+# Request for Comments (RFC)
+
+This directory hosts proposals (RFCs) of evolutionary topics around VAST. The
+objective is to actively include the community in the discussion.
+
+## Add a new RFC
+
+The workflow to add a new RFC works as follows:
+
+1. Copy the template and increase the running counter:
+   ```bash
+   # Assumption: the last proposal is 041-proposal.md
+   cp 000-TEMPLATE.md 042-proposal-name.md
+   ```
+
+2. Write the proposal in Github Markdown.
+
+3. [Submit a pull request](https://github.com/tenzir/vast/pulls).
```

Comment:
nit: s/Submit/Open/?
The (too?) subtle difference between submitting and merging makes it unclear to me.

Reply:
Good idea.